packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```diff
@@ -14,6 +14,7 @@ import {
   ResolvedConfig,
   Connect,
 } from 'vite';
+import { encapsulateStyle } from '@angular/compiler';
```

Comment:
Do you know what version this was introduced in? We support Angular v17-19

Reply:
Looks like it was added in `17.1.0`: https://github.com/angular/angular/commit/f0377d33a1fc1e5ce304a47adf1bf917094cb2e0

I can just make it lazy as we only need it if liveReload is enabled, which will only work in v19+

Reply:
Good idea